recipe/meta.yaml

```diff
@@ -11,8 +11,10 @@ source:
   sha256: 75bf6a04b965de214332b3c6ae77332d96142a8a780d5622372b10f19e1885b3
 
 build:
-  number: 0
+  number: 1
   noarch: python
+  entry_points:
+    - queue-monitor = bluesky_widgets.apps.queue_monitor.main:main
```

Comment:
We probably can add a test in the `commands` section to check the binary exists (such as `which queue-monitor` or `queue-monitor --help`).